bundler/spec/spec_helper.rb

```diff
@@ -14,6 +14,11 @@ def bundler_2_available?
   ENV["SUITE_NAME"] == "bundler2"
 end
 
+# Load project files prepended with the bundler version, which is currently only ever bundler1
+def bundler_project_dependency_files(project)
+  project_dependency_files(File.join("bundler1", project))
+end
```

Comment:
This method provides a forward way of loading any fixtures modified in this PR, I left any existing calls to `project_dependency_files` alone to avoid bloating the diff.